game/templates/game/level_moderation.html

```diff
@@ -110,12 +110,13 @@
 
 {% if levelData %}
   <div class="tableWrapper">
-    {% if thead %}
     <table id="" class="CSSTable wide alternate smaller">
       <tr>
-      {% for header in thead %}
-        <th> {{header}} </th>
-      {% endfor %}
+        <th>{% trans "Student" %}</th>
+        <th>{% trans "Level name" %}</th>
+        <th>{% trans "Shared with" %}</th>
+        <th>{% trans "Play" %}</th>
+        <th>{% trans "Delete" %}</th>
```

Comment:
Can do.. - you could just wrap the text in the python source with the `ugettext` call: https://docs.djangoproject.com/en/1.9/topics/i18n/translation/#standard-translation